src/lib/Behat/PageElement/Fields/Country.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzPlatformAdminUi\Behat\PageElement\Fields;
+
+use EzSystems\EzPlatformAdminUi\Behat\Helper\UtilityContext;
+use PHPUnit\Framework\Assert;
+
+class Country extends EzFieldElement
+{
+    /** @var string Name by which Element is recognised */
+    public const ELEMENT_NAME = 'Country';
+
+    public function __construct(UtilityContext $context, string $locator, string $label)
+    {
+        parent::__construct($context, $locator, $label);
+        $this->fields['fieldInput'] = 'select';
+    }
+
+    public function setValue(array $parameters): void
+    {
+        $fieldInput = $this->context->findElement(
+            sprintf('%s %s', $this->fields['fieldContainer'], $this->fields['fieldInput'])
+        );
+
+        Assert::assertNotNull($fieldInput, sprintf('Input for field %s not found.', $this->label));
+
+        $fieldInput->selectOption($parameters['value']);
+    }
+
+    public function getValue(): array
+    {
+        $fieldInput = $this->context->findElement(
+            sprintf('%s %s', $this->fields['fieldContainer'], $this->fields['fieldInput'])
+        );
+
+        Assert::assertNotNull($fieldInput, sprintf('Input for field %s not found.', $this->label));
+
+        return [$fieldInput->getValue()];
+    }
+
+    public function verifyValue(array $value): void
```

Comment:
Just as a suggestion: `verifyValue` looks the same in many classes, it could be the default implementation of EzFieldElement (but once it's done it can stay this way).